src/utils/dato-utils.ts

```diff
@@ -234,3 +234,31 @@ export function hentSkjermetInfo(
         };
     }
 }
+
+export function hentSikkerhetsTiltakInfo(
+    harSikkerhetsTiltak: boolean | undefined,
+    tiltakBeskrivelse: string | undefined,
+    tiltakFra: string | undefined,
+    tiltakTil: string | undefined
+): SikkerhetsTiltakEtikettConfig {
+    if (!harSikkerhetsTiltak) {
+        return {
+            hidden: true,
+            tittel: null,
+            type: 'info'
+        };
+    }
+
+    var tittelVerdi = 'Sikkerhetstiltak';
```

Comment:
In general `let` is preferred over `var`, since variable-scoping is more intuitive with `let`.
```suggestion
    let tittelVerdi = 'Sikkerhetstiltak';
```